tests/ert/unit_tests/scheduler/test_openpbs_driver.py

```diff
@@ -302,86 +254,33 @@ async def started(iens):
     assert was_started == started_expected
 
 
-@given(words, st.integers(min_value=1), words)
+@given(st.integers(), st.integers(min_value=1), words)
 @pytest.mark.usefixtures("capturing_qsub")
-async def test_full_resource_string(memory_per_job, num_cpu, cluster_label):
+async def test_full_resource_string(realization_memory, num_cpu, cluster_label):
```

Comment:
Is the name still valid though? Since this one tests only realization memory in the resource string